### ISSUE TITLE
Unable to build for iOS

### ISSUE THREAD

**Issue by manification10** [chenjd/Unity-ARFoundation-HandDetection#2]
Is there any setting that needs to be changed to be built for iOS?

**Comment by chenjd** [comment]
Hi @manification10 . I test it today and it can be built to iOS ( via Xcode 11.2). I think you should modify the Development Team item in Build Setting editor.

**Comment by iamblesson** [comment]
Hi Chenjd,
I updated the project to Unity 2019.2.6f.
Built the project successfully.
However when I tried to run it on Xcode 11.0 I faced an error saying that there are some swift library files created with Swift 3.0 and they are not compatible with Xcode 11.0.

Resolved this issue with some help i found online.

But then faced another error which had something to do with the UnityFramework.h file.

Would it be possible to update this project to run with Unity 2019.2 and Xcode 11.0 and above? :)
Would be really helpful.

I am looking forward to building applications integrating CoreML with ARFoundation and this repo will be very helpful in acting as a template to help develop the same.
